From a7ac8a08dcd29e6fc2128b82af588571c994c351 Mon Sep 17 00:00:00 2001
From: leeffo <leeffo@yeah.net>
Date: Wed, 15 Mar 2023 13:19:35 +0800
Subject: [PATCH] feat: fix strip

---
 src/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 4942abb..5df6473 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -3,8 +3,8 @@ set(CMAKE_CXX_STANDARD_REQUIRED True)
 
 set(APP_NAME ${CMAKE_PROJECT_NAME})
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall ")
-set(CMAKE_CXX_FLAGS "-Wl,--as-need -fPIE -fpermissive")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -s")
+set(CMAKE_CXX_FLAGS "-Wl,--as-need -fPIE -fpermissive -s")
 set(CMAKE_EXE_LINKER_FLAGS "-pie")
 if (${CMAKE_SYSTEM_PROCESSOR} MATCHES "sw_64")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mieee")
-- 
2.20.1

